.development/googlesheets_dataclean.R

```diff
@@ -4,16 +4,28 @@ library(janitor)
 library(tidyr)
 library(googlesheets4)
 
-sheet_url <- "https://docs.google.com/spreadsheets/d/1usk9Q-5lA4bL_z6KXpUohc_2x_KhDgLxtm-YEtim_yk/"
+sheet_url <- "https://docs.google.com/spreadsheets/d/1007RLMHSukSJ5OfCcDJdnJW5QMZyS2P-81fe7utCZwk/"
 
 # authorise Google account via prompt
 gs4_auth()
 
 # define google sheets ----------------------------------------------------
-list_arguments <- list(sheet = c("Central & Western", "Wan Chai", "Eastern", "Southern"))
+dc_sheet_names <- sheet_names(sheet_url)
+dc_sheet_names2 <- dc_sheet_names[!grepl(x = dc_sheet_names, pattern = "^Master$|^DistrictCouncilKey$")]
```

Comment:
Could you add a comment to say what this regex is doing?

Reply:
Good shout, I'll add that